### PR TITLE
MOO-75, MOO-361: Make sure that backup files created are zipped with a password

### PR DESCRIPTION
**Pull request by edrisse**

### CODE REVIEW

File: backup/docker-compose/init.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+if [ ! -z "$1" ]
+then
+rm docker-compose*
+wget https://raw.githubusercontent.com/edrisse/openmrs-module-msfcore/master/backup/docker-compose/docker-compose.yml
```
Comment (edrisse) [inline backup/docker-compose/init.sh]: @lauravignoli we will also need to change this URLs once you approve the PR to point to Jembi repository

**Comment by lauravignoli** [comment]
Let's also implement a cron job for deleting files older than a week

File: backup/docker-compose/init.sh [hunk 2]
```diff
@@ -1,4 +1,14 @@
...
+wget https://raw.githubusercontent.com/edrisse/openmrs-module-msfcore/master/backup/docker-compose/docker-compose.override.yml
+wget https://raw.githubusercontent.com/edrisse/openmrs-module-msfcore/master/backup/docker-compose/docker-compose.prod.yml
+wget https://raw.githubusercontent.com/edrisse/openmrs-module-msfcore/master/backup/docker-compose/zip_file_password.txt
+echo $1 > zip_file_password.txt
+mkdir backup
+else
+  echo "Please specify a password for the zip file as command line argument"
```
[the remark this answers is not in the document]
Comment (edrisse) [reply backup/docker-compose/init.sh]: That's absolutely right. Will work on it.

**Comment by edrisse** [comment]
OK @lauravignoli , will also work on deleting old backups

**Comment by edrisse** [comment]
Performed the changes. Please review again. You will need to recreate your containers, I published a new image of `jembi/openmrs-module-msfcore`

**Pull request merged**